src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -420,7 +420,9 @@ namespace winrt::TerminalApp::implementation
                     // GH#11356 - we can't use the UWP apis for writing the file,
                     // because they don't work elevated (shocker) So just use the
                     // shell32 file picker manually.
-                    path = co_await SaveFilePicker(*_hostingHwnd, [control](auto&& dialog) {
+                    std::wstring filename{ tab.Title() };
+                    filename = til::clean_filename(filename);
+                    path = co_await SaveFilePicker(*_hostingHwnd, [filename](auto&& dialog) {

```

Comment:
```suggestion
                    path = co_await SaveFilePicker(*_hostingHwnd, [filename = std::move(filename)](auto&& dialog) {
```

This should compile as of C++14 and will remove an extra/unnecessary copy. However, it also doesn't matter since this is a very rarely-exercised codepath in a user interactive scenario.